docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Gift Cards
+---
+
+:::note
+Gift Cards are available in Saleor 3.1 (alpha stage). For production purposes, we recommend using the latest beta version of Saleor 3.0.
+:::
+
+This page contains information on how to manage gift cards by selecting a specific currency, sending them to customers, or setting expiring dates.
+
+## Introduction
+Gift cards are timeless and universal gifts. They are suitable for customers, employees, CEOs, business partners, or family members. Issuing gift cards can increase your sales and provide a better customer experience.
+
+## How to
+### How to create a gift card?
+Once you have created a product type that can be used as a gift card or voucher, you can issue a gift card. Go to the Gift Card page of the Catalog section. 
+
+![Gift cards](../screenshots/catalog-giftcards-main.png)
+
+Then click the Issue Card button on the top right corner of the page. A pop-up page will appear where you can carry out the following actions related to the gift card creation:  
+
+- Choose currency  
+
+- Enter the amount  
+
+- Send gift cards to customers  
+
+- Set expiry date  
+
+- Write notes
+
+- Assign activation requirement
+
+![Gift cards](../screenshots/catalog-giftcards-new.png)
+
+Once a gift card has been created you can manage its balance on the top of the gift card details page. You can also add a card tag (optional) and set an expiration date.  
+
+On the right side of the gift card details page, you can find basic information about the gift card such as the creation date.
+
+### How to set an automatic expiration for gift cards
+Click the three dots button on the top right corner of the page, near the Issue Card button. Click *Settings* and then you can set gift cards to expire after a certain time after their purchase. Remember that in some countries, gift cards expiry is prohibited by law.
+
+![Gift cards](../screenshots/catalog-giftcards-expiration.png)
+
+### How to manage metadata in the gift card details page
+Once a gift card has been created you can add private and public metadata for each gift card in the gift card details page. Use the button Add Field to add a new metadata field.
+
+![Gift cards](../screenshots/catalog-giftcards-metadata.png)
+
+### How to deactive a gift card
+Select a specific previously created gift card from the gift cards list and then click *deactivate*. 
```

Comment:
Since we mention the "deactivate" action we can write about the bulk actions on the gift card list page. User can select many gift cards, delete, activate, deactivate them from the list page. Also, all these actions can be performed in the details view.